cylc/flow/cfgspec/workflow.py

```diff
@@ -1730,57 +1731,33 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                 ''')
 
             with Conf('workflow state polling', desc=f'''
-                Configure automatic workflow polling tasks as described in
-                :ref:`WorkflowStatePolling`.
-
-                The items in this section reflect
-                options and defaults of the ``cylc workflow-state`` command,
-                except that the target workflow ID and the
-                ``--task``, ``--cycle``, and ``--status`` options are
-                taken from the graph notation.
+                Deprecated support for automatic workflow state polling tasks
+                as described in :ref:`WorkflowStatePolling`. Note the Cylc 7
+                "user" and "host" config items are not supported.
 
                 .. versionchanged:: 8.0.0
 
                    {REPLACES}``[runtime][<namespace>]suite state polling``.
-            '''):
-                Conf('user', VDR.V_STRING, desc='''
-                    Username of your account on the workflow host.
 
-                    The polling
-                    ``cylc workflow-state`` command will be
-                    run on the remote account.
-                ''')
-                Conf('host', VDR.V_STRING, desc='''
-                    The hostname of the target workflow.
+                .. deprecated:: 8.3.0
 
-                    The polling
-                    ``cylc workflow-state`` command will be run there.
-                ''')
+                   Please switch to workflow_state xtriggers.
```

Comment:
```suggestion
                   Please switch to :ref:`workflow_state xtriggers <Built-in Workflow State Triggers>`.
```
Or whatever makes Sphinx and Flake8 happy